services/galley/src/Galley/Intra/Client.hs

```diff
@@ -183,6 +187,28 @@ getClientByKeyPackageRef ref = do
     then Just <$> parseResponse (mkError status502 "server-error") r
     else pure Nothing
 
+-- | Calls 'Brig.API.Internal.getConvIdByKeyPackageRef'.
+getConvIdByKeyPackageRef :: KeyPackageRef -> App (Maybe (Qualified ConvId))
+getConvIdByKeyPackageRef ref = do
+  r <-
+    call Brig $
+      method GET
+        . paths ["i", "mls", "key-packages", toByteString' $ toUrlPiece ref, "conversation"]
+        . expectStatus (flip elem [200, 404])
+  if statusCode (responseStatus r) == 200
+    then Just <$> parseResponse (ErrorCall . LText.unpack) r
+    else pure Nothing
+
+-- | Calls 'Brig.API.Internal.getConvIdByKeyPackageRef'.
```

Comment:
The copy-pasta left-over: update the reference so it's _put_ instead of _get_. While at it, can you also please fix a typo in the `Wire.API.Routes.Internal.Brig` module where on line 210: there's this?
```haskell
  RespondEmpty 204 "Converstaion associated"                                                                                                                       
```

Reply:
Looks like a common typo; fixed 5-6 other occurrences, too.